### DISCUSSION TITLE
GNU gettext and *.desktop files?

### DISCUSSION THREAD

**Discussion by buhtz** [WeblateOrg/weblate#10982] [Q&A]
Does anyone know if I can somehow add *.desktop files as a translation component to Weblate when GNU gettext is involved?

**Comment by nijel** [comment]
You can use intltool to localize desktop files.

**Reply by buhtz** [reply]
Seems not to be part of GNU gettext.
Docu is very semi. Manpages do not help. Took me also some time to realize that "intltools" is not a command and there is no manpage. intltools-prepare polutes my repo with unecessary files.
There is no docu describing the whole workflow and how all these tools working together.

I would like to stick with a pure GNU gettext solution if possible.

**Reply by nijel** [reply]
You need something capable of building translated desktop files, GNU gettext itself cannot do that.

**Reply by emmapeel2** [reply]
po4a maybe?
